cmd/restore.go

```diff
@@ -384,6 +397,101 @@ func newTableRestoreCommand() *cobra.Command {
 	return command
 }
 
+func newRawRestoreCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "raw",
+		Short: "restore a raw kv range",
+		RunE: func(cmd *cobra.Command, _ []string) error {
```

Comment:
The cobra commands should just handle cli options and then call a function. Then it will be easy to create other interfaces to br or to use as a library.

Reply:
Sorry I don't know. I just tried to keep my code consistent with other commands here. @5kbpers Do you have any idea?